docs/src/main/asciidoc/using-vertx.adoc

```diff
@@ -19,15 +19,15 @@ If you are creating a new project, set the `extensions` parameter are follows:
 mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -DprojectGroupId=org.acme \
     -DprojectArtifactId=vertx-quickstart \
-    -Dextensions="vertx"
+    -Dextensions="vert.x"
```

Comment:
Wondering if it's a good idea to have the dot here? I suppose not having the dot would work anyway as you have an exact match for `quarkus-vertx`?

Reply:
The real/official name includes the dot. That's why. @vietj WDYT?

Reply:
Just removed the short name as it's not required anymore.

This case is handled by the partial name match.